validator/main.c

```diff
@@ -0,0 +1,44 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include "standalone/api.h"
+
+static void print_log(void* obj, int level, char* str)
+{
+    if (str != NULL) {
+        puts(str);
+    }
+}
+
+static void exit_with_error(const char* msg)
+{
+    puts(msg);
+    exit(EXIT_FAILURE);
+}
+
+int main(int argc, char* argv[])
```

Comment:
since this is standalone, could we add a test for it?

Reply:
Yes, this will be done by integrating with tenant unix and onebox tests.